app/views/bookings/_form.html.erb

```diff
@@ -0,0 +1,38 @@
+<%= form_for([:room, @booking]) do |f| %>
+  <div class="form-row">
+    <div class="form-group col-md-6">
+      <%= f.label :start_date, t("date_start"), class: "label-booking" %>
+      <%= f.date_field :start_date, value: @date_start, class: "form-control" %>
+      <%= render "shared/error_messages", object: f.object , attr: :start_date %>
+    </div>
+    <div class="form-group col-md-6">
+      <%= f.label :end_date, t("end_start"), class: "label-booking" %>
+      <%= f.date_field :end_date, value: @date_end, class: "form-control" %>
+      <%= render "shared/error_messages", object: f.object , attr: :end_date %>
+    </div>
+  </div>
+  <%= f.fields_for :customer do |add| %>
+    <div class="form-group">
+      <%= add.label :name, t("name_cus"), class: "label-booking" %>
+      <%= add.text_field :name, class: "form-control" %>
+    </div>
+    <%= render "shared/error_messages", object: add.object , attr: :name %>
+    <div class="form-group">
+      <%= add.label :birthday, t("birthday"), class: "label-booking" %>
+      <%= add.date_field :birthday, class: "form-control" %>
+    </div>
+    <%= render "shared/error_messages", object: add.object , attr: :birthday %>
+    <div class="form-group">
+      <%= add.label :address, t("address"), class: "label-booking" %>
+      <%= add.text_field :address, class: "form-control" %>
+    </div>
+    <%= render "shared/error_messages", object: add.object , attr: :address %>
+    <div class="form-group">
+      <%= add.label :id_card, t("id_card"), class: "label-booking" %>
+      <%= add.text_field :id_card, class: "form-control" %>
+    </div>
+    <%= render "shared/error_messages", object: add.object , attr: :id_card %>
+  <% end %>
+  <%= f.hidden_field :room_id, value: @room.id%>
```

Comment:
Đâu cần phải truyền hidden file anh ơi, chỉ cần get trên url là dc mà
`rooms/room_id/bookings/`

Reply:
em hiểu mà, cơ mà cái này truyền trong hiddent field thì vẫn có thể bị hack dữ liệu, và dữ liệu này lại dc check thì nó ko an toàn để lưu vào DB nhé

Reply:
a sửa rồi á em ơi. a dùng merge params rồi ^^